qiskit_rigetti/_qcs_backend.py

```diff
@@ -143,6 +143,10 @@ def run(
         if not isinstance(run_input, list):
             run_input = [run_input]
 
+        bindings = options.get("parameter_binds") or []
```

Comment:
```suggestion
        bindings = options.get("parameter_binds", [])
```

Reply:
Unless `parameter_binds` could potentially be set but `None`?

Reply:
I had that and apparently it ends up as `None` if not present and the default doesn't apply